js/gdm/authPrompt.js

```diff
@@ -636,7 +659,7 @@ var AuthPrompt = class {
         this.setQuestion(_("Enter unlock code provided by customer support:"));
         this.setMessage(
             // Translators: Password reset. The first %s is a verification code and the second is an email.
-            _("Please inform customer support of your verification code %s by emailing %s. The code will remain valid until you click Cancel or turn off your computer.").format(
+            _("Please inform customer support of your verification code %s by emailing %s. The code will remain valid until your next login.").format(
```

Comment:
```suggestion
            _("Please inform customer support of your verification code %s by emailing %s. They will provide you with an unlock code.").format(
```

I don't think we need to mention how long the unlock code remains valid if it's (for all intents and purposes) "as long as necessary".

Reply:
Done.